build/Shared/IsExternalInit.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if NETFRAMEWORK || NETSTANDARD
+#if !NET5_0_OR_GREATER
```

Comment:
One of the test projects targeting netcoreapp3.1 started failing once I added the `RequiredModiferAttributes` file, hence I needed to fix this, as this `IsExernalInit` attribute was added in .NET 5.